examples/with-supertokens/pages/auth/[[...path]].js

```diff
@@ -18,15 +18,17 @@ export default function Auth() {
   }, [])
 
   return (
-    <div className={styles.container}>
-      <Head>
-        <title>SuperTokens 💫</title>
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
+    <SuperTokensWrapper>
```

Comment:
Why do we need this here? We are surrounding the _app.js component with `SuperTokensWrapper` anyway?